content/telegraf/v1.24/configure_plugins/custom_builder.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Reduce Telegraf binary size with customized agent
+description: Use the custom builder tool to reduce Telegraf binary disk space. 
+menu:
+  telegraf_1_24:
+
+    name: Reduce binary size
+    Parent: Configure plugins
+    weight: 20
```

Comment:
This shouldn't be in configuring plugins. In the process of setting Telegraf up, this should come before installing the binary. I think we need to turn the current Install doc into a section and add this page under Install.

```suggestion
menu:
  telegraf_1_24:
    name: Custom-compile Telegraf
    Parent: Install
weight: 20
```

Reply:
so maybe?:

To install Telegraf, do the following:
1. [Download Telegraf](https://docs.influxdata.com/telegraf/v1.24/install/#download)
2. [Review requirements](https://docs.influxdata.com/telegraf/v1.24/install/#requirements)
3. (Optional?) [Custom-compile Telegraf](insert link to new section) 
     **Tip:** Complete this step to reduce the size of your Telegraf binary.
4. [Install](https://docs.influxdata.com/telegraf/v1.24/install/#installation)

Reply:
Yeah this makes sense to put it somewhere near install. People will most likely try downloading Telegraf out-of-the-box first then reference something like this for building something customized.